activefires_pp/spatiotemporal_alarm_filtering.py

```diff
@@ -203,6 +210,11 @@ def close(self):
                 LOG.exception("Couldn't stop publisher.")
 
 
+def get_xauth_environment_variable():
+    """Get the environment variable for the X-Authentication-token needed for posting to the API."""
+    return os.environ.get('XAUTH_SMHI_FIREALARMS_REST_API')
```

Comment:
`SMHI` should be removed from the variable name.

Reply:
Fixed.